packages/grant-explorer/package.json

```diff
@@ -117,12 +125,14 @@
     "@testing-library/dom": ">=7.21.4 ",
     "@types/dompurify": "^2.4.0",
     "@types/jest": "^27.4.1",
+    "@types/markdown-it": "^12.2.3",
     "@types/node": "^17.0.25",
     "@types/react": "^18.0.6",
     "@types/react-dom": "^18.0.2",
     "@types/react-gtm-module": "^2.0.1",
     "@types/testing-library__jest-dom": "^5.14.5",
     "autoprefixer": "^10.4.7",
+    "craco-esbuild": "0.5.2",
```

Comment:
Do we need this ?

Reply:
this is used in our build, so very much needed 😃